dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.org.eclipse.microprofile.faulttolerance-4.1.feature

```diff
@@ -0,0 +1,10 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.org.eclipse.microprofile.faulttolerance-4.1
+singleton=true
+# io.openliberty.mpCompatible-x.x comes from io.openliberty.microprofile.cdi.api features
+-features=io.openliberty.microprofile.cdi.api-4.0; ibm.tolerates:="4.1"
```

Comment:
This appears to be wrong.  io.openliberty.microprofile.cdi.api-4.0 has mpCompatible 6.0 and 6.1 toleration, which isn't appropriate for fault tolerance 4.1.

Reply:
io.openliberty.microprofile.cdi.api-4.0 had mpCompatible-7.0 toleration added